samples/nrf_rpc/entropy_nrf53/sysbuild.cmake

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+get_property(PM_DOMAINS GLOBAL PROPERTY PM_DOMAINS)
+
+# Include app core image if enabled
+if(SB_CONFIG_SOC_NRF5340_CPUAPP)
+  # Get application core board target
+  string(REPLACE "/" ";" split_board_qualifiers "${BOARD_QUALIFIERS}")
+  list(GET split_board_qualifiers 1 target_soc)
+  set(board_target_netcore "${BOARD}/${target_soc}/cpunet")
+  set(target_soc)
+
+  ExternalZephyrProject_Add(
+    APPLICATION remote
+    SOURCE_DIR ${APP_DIR}/remote
+    BOARD ${board_target_netcore}
```

Comment:
Cannot just hardcode target name here? nRF5340 has only one remote target

Reply:
This is the proper way to do it to allow users to see how to add targets properly and to allow adding additional targets in future if needed